From a738a2c11229b464329c02866503a520ff567483 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 25 Oct 2023 18:16:10 +0200
Subject: [PATCH 0933/1141] serial: stm32: enable SERIAL_STM32_CONSOLE on every
 state of SERIAL_STM32

SERIAL_STM32_CONSOLE is possible even if SERIAL_STM32 is loaded in module.
So change the dependency.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: If7c87a37a9ec8f61234f113d578b29c6ee7a14c6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/336270
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/tty/serial/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/tty/serial/Kconfig
+++ b/drivers/tty/serial/Kconfig
@@ -1410,7 +1410,7 @@ config SERIAL_STM32
 
 config SERIAL_STM32_CONSOLE
 	bool "Support for console on STM32"
-	depends on SERIAL_STM32=y
+	depends on SERIAL_STM32
 	select SERIAL_CORE_CONSOLE
 	select SERIAL_EARLYCON
 
